src/core/a-animation.js

```diff
@@ -363,6 +364,8 @@ function getAnimationValues (el, attribute, dataFrom, dataTo, currentValue) {
     getForCoordinateComponent();
   } else if (['true', 'false'].indexOf(dataTo) !== -1) {
     getForBoolean();
+  } else if (dataTo && dataTo[0] === '#') {
```

Comment:
Good idea.  Can you point me to the right direction on this?  Is there already a currently existing way to check if these are valid colors or would this be something I need to add?